sdk/servicebus/Azure.Messaging.ServiceBus/tests/Sender/SenderLiveTests.cs

```diff
@@ -160,9 +160,9 @@ public async Task ClientProperties()
         {
             await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false))
             {
-                await using var sender = new ServiceBusClient(TestEnvironment.ServiceBusConnectionString).CreateSender(scope.QueueName);
+                await using var sender = new ServiceBusClient(ServiceBusTestEnvironment.Instance.ServiceBusConnectionString).CreateSender(scope.QueueName);
                 Assert.AreEqual(scope.QueueName, sender.EntityPath);
-                Assert.AreEqual(TestEnvironment.FullyQualifiedNamespace, sender.FullyQualifiedNamespace);
+                Assert.AreEqual(ServiceBusTestEnvironment.Instance.FullyQualifiedNamespace, sender.FullyQualifiedNamespace);
```

Comment:
Would it be possible to add a TestEnvironment property to ServiceBusLiveTestBase so that we don't have to use ServiceBusTestEnvironment.Instance everywhere?